azure-http/sample.json

```diff
@@ -19,7 +19,7 @@
          "Gateway 32-Bit",
          "Gateway 64-Bit"
       ],
-      "os": ["yocto", "wrs_32", "wrs_64"]
+      "os": ["yocto", "wrs_32"]
```

Comment:
Shahab, just for my knowledge...why was wrs_64 removed?

Reply:
In wrs_64, Azure IoT library cannot currently be installed since one of the dependencies is not signed.

Reply:
Okay. Is that something azure would do? Is there a JIRA to track that?

Reply:
I'm not sure who should fix it, but I'll follow up in a separate task.